gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java

```diff
@@ -193,7 +193,7 @@ protected void configureForSourceSet(final SourceSet sourceSet, SpotBugsTask tas
             public FileCollection call() {
                 // the simple "classes = sourceSet.output" may lead to non-existing resources directory
                 // being passed to SpotBugs Ant task, resulting in an error
-                return project.fileTree(sourceSet.getOutput().getClassesDirs()).builtBy(sourceSet.getOutput());
+                return project.fileTree(sourceSet.getOutput().getClassesDir()).builtBy(sourceSet.getOutput());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getClassesDir"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)